service/internal/security/hsm.go

```diff
@@ -237,6 +238,77 @@ func lookupSlotWithLabel(ctx *pkcs11.Ctx, label string) (uint, error) {
 	return 0, ErrHSMUnexpected
 }
 
+func New(c *HSMConfig) (*HSMSession, error) {
```

Comment:
Is there a reason for a second New hsm func here?

Reply:
Nope...  Merge error.  Removed...